rust/libnewsboat/src/keymap.rs

```diff
@@ -0,0 +1,288 @@
+use nom::{
+    branch::alt,
+    bytes::complete::{escaped_transform, is_not, tag, take},
+    character::complete::{none_of, space0},
+    combinator::{complete, eof, map, recognize, value},
+    multi::{many0, many1, separated_list0, separated_list1},
+    sequence::{delimited, tuple},
+    IResult,
+};
+
+fn unquoted_token(input: &str) -> IResult<&str, String> {
+    let parser = tuple((none_of("\";"), is_not(" ;")));
```

Comment:
This does not allow single-character unquoted tokens.
Can be reproduced with a test case like:
```
    #[test]
    fn t_tokenize_operation_sequence_allows_single_character_unquoted() {
        assert_eq!(
            tokenize_operation_sequence(r#"set a b"#).unwrap(),
            vec![vec!["set", "a", "b"]]
        );
    }
```

Reply:
Right, thanks! I added a corresponding C++ test in https://github.com/newsboat/newsboat/pull/1203/commits/b8f019bbd4409850b86a89ed710431e13d247bbc (it passes on master too), then fixed the bug in https://github.com/newsboat/newsboat/pull/1203/commits/3a5a76de6c4093637c3f8944b632bb1bffd3a4a1.